libraries/AP_HAL_SITL/SITL_cmdline.cpp

```diff
@@ -241,6 +314,15 @@ void SITL_State::_parse_command_line(int argc, char * const argv[])
             break;
         case CMDLINE_FGVIEW:
             _use_fg_view = true;
+        case CMDLINE_GAZEBO_ADDRESS:
+            _gazebo_address = gopt.optarg;
+            break;
+        case CMDLINE_GAZEBO_PORT_IN:
+            _gazebo_port_in = atoi(gopt.optarg);
```

Comment:
hummm error here... defining instance and gazebo port only use the lastest parameter use ... should be exclusif !